StartScreen/Javascript.js

```diff
@@ -1,23 +1,40 @@
 function region(){
-document.getElementsByClassName("regionSelect")[1].click();	}
+document.getElementsByClassName("regionSelect")[1].click();	
+}
  region();
 function tabs(){
-document.getElementsByClassName("strong tab")[0].click();		}
+document.getElementsByClassName("strong tab")[0].click();		
+}
 tabs();
 function get(){

```

Comment:
really like the conjunction of the function to the task it is doing. The rest of them are well named function except I was just a little confused on get as function name, maybe OSBlock? TileofTab? not sure just that all of them seem well titled except this one.

Reply:
I'll change it to TileTabs just to make sure it is known what I am changing.